packetnetworking/cli.py

```diff
@@ -68,6 +69,37 @@ def cli(
             )
         )
 
+    # arbitrary retry count
+    max_attempts = 10
```

Comment:
It might be nice to have a command line configureable options for setting this. I think the hope is others may be able to implement this tool in their own workflows, which could potentially be calling out to a self hosted metadata service. So allowing them to modify this value or potentially disable retires easily might be nice.

Reply:
Sure, that sounds reasonable -- think this is an acceptable default value?

Reply:
Command-line option is a great idea but feels a little beyond the scope of improving the state of affairs. Can we allow anyone who wants to tune the parameter to add the feature?

Reply:
Since this feature was just being added, I feel now is a great time to add it. Especially with such a simple change.

@zevweiss yup, I think 10 is a great default value!

Reply:
@zevweiss looks like you may have missed using the new variable. Can probably remove max_attempts defined here, and rename max_att to max_attempts